keyboards/whitefox/keymaps/constexpr/rules.mk

```diff
@@ -0,0 +1,8 @@
+BOOTMAGIC_ENABLE = no
+CONSOLE_ENABLE = yes
+TAP_DANCE_ENABLE = yes
+UNICODE_ENABLE = yes
+
+ifndef QUANTUM_DIR
```

Comment:
This block is a hold over from an older makefile configuration.  If you could remove it?

Reply:
I was wondering about this. Sure thing, I'll do that right away.